.github/CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
 
 ### Improvements
 
+* Update plugin test matrix GPU workflow names.
+  [(#918)](https://github.com/PennyLaneAI/pennylane-lightning/pull/918)
+
```

Comment:
This has to be technically fixed in https://github.com/PennyLaneAI/pennylane-lightning/pull/911! We don't need two entrypoints in the changelog for a single task.